locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -1647,15 +1630,17 @@ msgstr "专业型"
 
 #: meta/views.py:259 meta/views.py:336
 msgid "The highest average number of takedowns"
-msgstr ""
+msgstr "平均击杀数最高"
 
+# Is this highest mean lowest?
+# What is the difference between kills and takedowns?
```

Comment:
@jeklau Takedowns = kills + assists. Kills is like last hit in other MOBAs. You can find usage of this string on https://masterleague.net/tournament/38/players/ The more takedowns did the player, the better.